### PR TITLE
update warnOldDocs.js and symlink for release

### PR DESCRIPTION
**Pull request by rkuhn**

### CODE REVIEW

**Comment by patriknw** [comment]
LGTM I guess

**Comment by rkuhn** [comment]
Don’t be so lax, JS needs review, too! :-)

File: akka-docs/_sphinx/themes/akka/static/warnOldDocs.js [hunk 1]
```diff
@@ -3,28 +3,67 @@ jQuery(document).ready(function ($) {
 });
 
 function initOldVersionWarnings($) {
-  $.get(versionsJsonUri(), function (akkaVersionsData) {
-    var thisVersion = browsedAkkaVersion();
-    if (thisVersion.includes("-SNAPSHOT")) {
+  $.get("//akka.io/versions.json", function (akkaVersionsData) {
```
Comment (ktoso) [inline akka-docs/_sphinx/themes/akka/static/warnOldDocs.js]: oh, interesting (the `//`)

File: project/scripts/release [hunk 2]
```diff
@@ -361,16 +361,13 @@ if [ $dry_run ]; then
...
 
-echolog "Updating http://doc.akka.io/docs/versions.json [from $script_dir/../../akka-docs/versions.json]..."
-if [ $dry_run ]; then
-  echodry "Not actually updating versions.json. Command:"
-  echodry "  important scp $script_dir/../../akka-docs/versions.json ${publish_path}/docs/"
-else
-  important scp $script_dir/../../akka-docs/versions.json ${publish_path}/docs/
-fi
+echolog "*****"
+echolog "Do not forget to update versions.json on akka.github.com!"
```
Comment (ktoso) [inline project/scripts/release]: Yeah, good

File: akka-docs/_sphinx/themes/akka/static/warnOldDocs.js [hunk 3]
```diff
@@ -36,17 +75,25 @@ function showVersionWarning(version, akkaVersionsData, seriesInfo) {
...
-            '</p>');
+            '<p><span style="font-weight: bold">This version of Akka (' + site.p + ' / ' + version + ') is outdated and not supported! </span></p>' +
+            '<p>Please upgrade to version <a href="' + insteadSeries + '">' + instead.latest + '</a> as soon as possible.</p>' +
+            '<p id="samePageLink"></p>');
+    $.ajax({
+      url: insteadPage,
+      type: 'HEAD',
+      success: function() {
+        $('#samePageLink').html('<a href="' + insteadPage + '">Click here to go to the same page on the ' + instead.latest + ' version of the docs.</a>');
+      }
```
Comment (ktoso) [inline akka-docs/_sphinx/themes/akka/static/warnOldDocs.js]: great! :-)

File: akka-docs/_sphinx/themes/akka/static/warnOldDocs.js [hunk 4]
```diff
@@ -3,28 +3,67 @@ jQuery(document).ready(function ($) {
...
+  } else {
+    var found = akkaVersionsData[project][instead];
+    var proj = project;
+  }
+  return {"latest":found.latest, "project":proj};
+}
+
+function targetUrl(samePage, site, instead) {
+  var page = site.r;
+  if (samePage !== true) {
```
Comment (ktoso) [inline akka-docs/_sphinx/themes/akka/static/warnOldDocs.js]: have to share this wonderful javascript programming pattern: `if (!!!samePage)`

Comment (ktoso) [inline akka-docs/_sphinx/themes/akka/static/warnOldDocs.js]: (keep as it, just sharing for fun (also works with undefined etc))

**Comment by ktoso** [comment]
Very good, LGTM

**Comment by rkuhn** [comment]
I added the warning for browsing the snapshot docs, recommending users to view `current` instead, and I also found and fixed a problem with cookie storage (since the same script is now used for both akka and akka-stream-and-http-experimental.

**Pull request merged**